lib/modem_key_mgmt/modem_key_mgmt.c

```diff
@@ -272,6 +272,42 @@ int modem_key_mgmt_delete(nrf_sec_tag_t sec_tag,
 	return 0;
 }
 
+int modem_key_mgmt_sectag_clear(nrf_sec_tag_t sec_tag)
+{
+	int err;
+	bool cmee_was_enabled;
+	char *token;
+	uint32_t tag, type;
+
+	cmee_enable(&cmee_was_enabled);
+
+	scratch_buf[0] = '\0';
```

Comment:
To set the scratch buffer to empty to avoid reading garbage if the read below (nrf_modem_at_cmd) fails. Though, as I added the return on error, it can be removed.